package.json

```diff
@@ -140,8 +139,7 @@
     "c8": "^7.4.0",
     "chalk": "^2.4.1",
     "chrome-devtools-frontend": "1.0.922924",
-    "conventional-changelog-cli": "^1.3.4",
```

Comment:
don't we use it here?

https://github.com/GoogleChrome/lighthouse/blob/d9e84e1d71e04ddae4f53d8e013683d6ca4c17ce/lighthouse-core/scripts/release/prepare-commit.sh#L44

Reply:
oh damn. okay yes.

okay instead i updated it to latest. also tested that it still works well.